src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -828,6 +834,53 @@ public void testSubmitLink() {
         feedbackPage = getFeedbackPageForInstructor(idOfInstructorWithSessions);
     }
     
+    public void testCopySessionModalButtons() {
+        feedbackPage.copyFeedbackSessionTestButtons("Session 1", newSession.courseId);
+        
+        assertFalse(feedbackPage.isCopySubmitButtonEnabled());
+        
+        ______TS("click on a row");
+        
+        feedbackPage.clickCopyTableAtRow(0);
+        assertTrue(feedbackPage.isRowSelected(0));
+        assertTrue(feedbackPage.isRadioButtonChecked(0));
+        assertTrue(feedbackPage.isCopySubmitButtonEnabled());
+        
+        ______TS("click on a radio button");
+        
+        feedbackPage.clickCopyTableRadioButtonAtRow(2);
+        assertTrue(feedbackPage.isRowSelected(2));
+        assertTrue(feedbackPage.isRadioButtonChecked(2));
+        assertFalse(feedbackPage.isRowSelected(0));
+        assertFalse(feedbackPage.isRadioButtonChecked(0));
+        assertTrue(feedbackPage.isCopySubmitButtonEnabled());
+        
+        ______TS("click on another radio button");
+        
+        feedbackPage.clickCopyTableRadioButtonAtRow(1);
+        assertTrue(feedbackPage.isRowSelected(1));
+        assertTrue(feedbackPage.isRadioButtonChecked(1));
+        assertFalse(feedbackPage.isRowSelected(2));
+        assertFalse(feedbackPage.isRadioButtonChecked(2));
+        assertTrue(feedbackPage.isCopySubmitButtonEnabled());
+        
+        ______TS("click on a row");
+        
+        feedbackPage.clickCopyTableAtRow(3);
+        assertTrue(feedbackPage.isRowSelected(3));
+        assertTrue(feedbackPage.isRadioButtonChecked(3));
+        assertFalse(feedbackPage.isRowSelected(1));
+        assertFalse(feedbackPage.isRadioButtonChecked(1));
+        assertTrue(feedbackPage.isCopySubmitButtonEnabled());
+        
+        ______TS("click on a radio button of the same row");
+        
+        feedbackPage.clickCopyTableRadioButtonAtRow(3);
+        assertTrue(feedbackPage.isRowSelected(3));
+        assertTrue(feedbackPage.isRadioButtonChecked(3));
+        assertTrue(feedbackPage.isCopySubmitButtonEnabled());
```

Comment:
Perhaps we can add just one more test -

Currently, it is checked that the copy submit button is enabled upon clicking on a row but not for a radio button, perhaps after all the tests above, we can add a page refresh and click on a radio button and ensure that the copy submit button is still enabled.